package.json

```diff
@@ -158,6 +158,6 @@
   },
   "types": "./dist/index.d.ts",
   "engines": {
-    "node": ">=12 <20.0.0"
+    "node": ">=12 <=20.0.0"
```

Comment:
Isn't it better to change it to `<=20`
Otherwise you say it's only working on 20.0.0, but not in 20.1.0 and 20.2.0, etc.

Reply:
Yes - that was addressed in #545 

https://github.com/cloudevents/sdk-javascript/blob/11442d32d307a0e8416ed573ce34fc825d3b63c4/package.json#L163

Reply:
Oh I see now... gotcha